dags/utils/django.py

```diff
@@ -222,3 +222,21 @@ def django_schema_create_and_check(schema_name: str, sql: str, dry_run=True) ->
     if schema_name not in tables_all:
         raise SystemError(f"Table pas crée malgré execution SQL OK: {schema_name}")
     logger.info(f"Création schema pour {schema_name=}: succès 🟢")
+
+
+def get_model_fields(model, with_relationships=True, latlong=False):
+    fields = []
+    for field in model._meta.get_fields():
+        if field.is_relation and not with_relationships:
+            continue
+        if field.one_to_many or field.many_to_many:
+            fields.append(field.name[0:-1] + "_codes")
+        elif field.many_to_one:
+            fields.append(field.name + "_code")
+        else:
+            fields.append(field.name)
+    if latlong:
+        fields.extend(["latitude", "longitude"])
+        if "location" in fields:
+            fields.remove("location")
+    return fields
```

Comment:
Je me dis qu'on aurait moyen de vérifier en fin de fonction que nos champs sont bien inclus dans le modèle en question. 

Soit via une méthode django, soit en python avec `collections.Counter`, genre 

```suggestion
from collections import Counter
    # ...
    if latlong:
        fields.extend(["latitude", "longitude"])
        if "location" in fields:
            fields.remove("location")
    if Counter(fields) - Counter(model._meta.get_fields()):
      raise ValueError(f"A field passed to get_model_fields is not included in the model {model} - {fields}")
    return fields
```

Reply:
je garde pour plus tard, peur de la régression